File: 58-length-of-last-word/length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
      lastLength = 0
      currLength = 0

      for char in s:
        if char != " ":
          currLength += 1
          lastLength = currLength
        else:
          currLength = 0
      
      return lastLength

File: 58-length-of-last-word/length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let lastLength = 0;
    let currLength = 0;

    for (let i = 0; i < s.length; i++) {
        if (s[i] !== " ") {
          currLength++;
          lastLength = currLength;
        } else {
          currLength = 0
        }
    }

    return lastLength
};